### PR TITLE
Refactor Code to Follow Proper Folder and Code Convention

### PR DESCRIPTION
**Pull request by raveracker**
- Used FC<Props> as applying props to Functional Component
- Implemented Proper Folder Structure for Containers and Components
- Removed Unwanted Codes
- Fix Header Responsive Issue on sm breakpoint

### CODE REVIEW

File: src/components/navigationBar.tsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import React, { FC } from 'react'
```
Comment (pallavisaraogi) [inline src/components/navigationBar.tsx:1]: the filename should be `kabab-case`

**Review by pallavisaraogi — CHANGES_REQUESTED** [review]

File: src/components/navigationBar.tsx (hunk 1 above)
Comment (raveracker) [reply src/components/navigationBar.tsx]: Done

File: src/containers/header/header.tsx [hunk 2]
```diff
@@ -1,7 +1,7 @@
-import React, { useEffect, useState } from 'react'
```
Comment (pallavisaraogi) [inline src/containers/header/header.tsx:1]: any specific reason for making folders for a single file?

Comment (raveracker) [reply src/containers/header/header.tsx:1]: Yeah, if one section has more than one container

Comment (pallavisaraogi) [reply src/containers/header/header.tsx:1]: and what's the difference btw container and component files?

Comment (raveracker) [reply src/containers/header/header.tsx]: Components are individual components, container is an entire section which contains components

**Pull request merged**